components/progressTracker/theme.css

```diff
@@ -0,0 +1,167 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-animations';
+
+.progress-tracker {
```

Comment:
No need to prefix your classnames as we are using modules anyway.

Reply:
Changed in the latest commit

 :ok: by @Kemosabert on 19/06/2018 11:40:42